preprocessing/tabular_data/training_model_on_tabular_data.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p>should be "an S3 bucket" - pease fix throughout. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>